tests/monitors/postgresql/postgresql_test.py

```diff
@@ -50,3 +58,25 @@ def test_postgresql():
         with run_agent(config) as [backend, _, _]:
             assert wait_for(p(has_datapoint_with_dim, backend, "plugin", "postgresql")), "Didn't get postgresql datapoints"
             assert wait_for(p(has_datapoint_with_metric_name, backend, "pg_blks.toast_hit"))
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_postgresql_in_k8s(agent_image, minikube, k8s_observer, k8s_test_timeout, k8s_namespace):
+    yaml = os.path.join(os.path.dirname(os.path.realpath(__file__)), "postgresql-k8s.yaml")
+    monitors = [
+        {"type": "collectd/postgresql",
+         "discoveryRule": get_discovery_rule(yaml, k8s_observer, namespace=k8s_namespace),
+         "databases": [{"name": "test", "username": "test_user", "password": "test_pwd"}],
+         "username": "test_user", "password": "test_pwd"},
```

Comment:
By default, `run_k8s_monitors_test` will check for the `testing123` password string in the agent status output. You should either change the postgresql password to `testing123` or add the `passwords=['test_pwd']` argument to `run_k8s_monitors_test`.

Reply:
Just wish to know, where does it impacts, right now i am not seeing any issues without passing passwords=['test_pwd'] as an argument

Reply:
Part of `run_k8s_monitors_test` is to test for cleartext passwords in the agent status output. Without the change, `run_k8s_monitors_test` will be testing for wrong the password, and if there are any changes to the postgresql monitor in the future that will expose the password, this test will not catch it.

Reply:
Ok. Will make the changes do necessary testing.